tensorflow/inference/docker/build_artifacts/sagemaker/serve.py

```diff
@@ -28,9 +28,7 @@
 JS_INVOCATIONS = "js_content tensorflowServing.invocations"
 GUNICORN_PING = "proxy_pass http://gunicorn_upstream/ping"
 GUNICORN_INVOCATIONS = "proxy_pass http://gunicorn_upstream/invocations"
-MULTI_MODEL = "s" if os.environ.get("SAGEMAKER_MULTI_MODEL", "False").lower() == "true" else ""
-MODEL_DIR = f"model{MULTI_MODEL}"
-CODE_DIR = "/opt/ml/{}/code".format(MODEL_DIR)
+CODE_DIR = "/opt/ml/code" if os.environ.get("SAGEMAKER_MULTI_MODEL", "False").lower() == "true" else "/opt/ml/model/code"
```

Comment:
thinking there is a better way to handle this as a bool, rather than string comparison. Se it was done this way before so not pushing to have changed on this PR, but something to look at on the next round.

Reply:
Yes we can update this is in a future PR.